lib/check_names.cc

```diff
@@ -2,21 +2,35 @@
 
 #include <prometheus/check_names.h>
 
+#if defined(__GLIBCXX__) && __GLIBCXX__ <= 20150623
```

Comment:
Unfortunately libstdc++ is not properly versioned. The date jumps back and forth for bugfix releases: [GCC releases](https://gcc.gnu.org/develop.html#timeline). I picked the value from latest GCC 4.8 on centos7.